app/controllers/submission.js

```diff
@@ -388,7 +396,16 @@ function curateGenesAndAnnotations(req, res) {
             });
 
             // Now that the request is valid, start the update.
-            return performSubmissionOrCuration(req, validationResult, curSubmission);
+            return performSubmissionOrCuration(req, validationResult, curSubmission).then(() => {
+
+                if (req.body.annotations.every(newAnnotation => {
+                    return newAnnotation.status != 'pending';
+                })) {
+                    //When we get emails, replace hard coded email with submitter email
+                    sendEmail.createMessage("ajr1644@rit.edu", "Cur");
+                }
+
+            });
```

Comment:
You need to forward the isCuration through to the next promise iirc.

Reply:
Addressed in upcoming push